packages/patternfly-4/react-core/src/components/Wizard/Wizard.tsx

```diff
@@ -44,6 +44,14 @@ export type WizardStepFunctionType = (newStep: { id?: string | number; name: str
 export interface WizardProps {
   /** True to show the wizard */
   isOpen?: boolean;
+  /** True to set full height wizard */
+  isFullHeight?: boolean;
+  /** True to set full width wizard */
+  isFullWidth?: boolean;
+  /** Default width of the Modal. */
```

Comment:
Can you update the comments here to say Wizard.

Reply:
Thanks!